README.md

```diff
@@ -325,7 +325,7 @@ can reuse existing local data when restarting.
 Volume | Kubernetes | PMEM-CSI | Limitations
 --- | --- | --- | ---
 Persistent | supported | supported | topology aware scheduling<sup>1</sup>
-Ephemeral | [in design](https://github.com/kubernetes/enhancements/blob/master/keps/sig-storage/20190122-csi-inline-volumes.md#proposal) | in design | topology aware scheduling<sup>1</sup>, resource constraints<sup>2</sup>
+Ephemeral | supported | supported | resource constraints<sup>2</sup>
```

Comment:
We probably should add to the Kubernetes column remarks about which versions support it, something like "alpha support in 1.15 (CSIInlineVolume), beta in 1.15".

Reply:
When looking at the rendered file, I noticed that the definition list (https://github.com/intel/pmem-csi#volume-persistency) is no longer rendered with the text on the next paragraph. This intentionally had two spaces (https://meta.stackexchange.com/questions/72395/is-it-possible-to-have-definition-lists-in-markdown/298872#298872) but we lost those in b30c9a051 - can you add them back?

Reply:
> When looking at the rendered file, I noticed that the definition list (https://github.com/intel/pmem-csi#volume-persistency) is no longer rendered with the text on the next paragraph. This intentionally had two spaces (https://meta.stackexchange.com/questions/72395/is-it-possible-to-have-definition-lists-in-markdown/298872#298872) but we lost those in [b30c9a0](https://github.com/intel/pmem-csi/commit/b30c9a0512420fd51e97013ad10ef64099373b44) - can you add them back?

Done.

Reply:
> We probably should add to the Kubernetes column remarks about which versions support it, something like "alpha support in 1.15 (CSIInlineVolume), beta in 1.15".

Added a column remark as suggested. Can you please check if it's good enough.